src/mongo/unittest/unittest.cpp

```diff
@@ -32,11 +32,13 @@
 
 #include "mongo/unittest/unittest.h"
 
-#include <iostream>
+//#include <fstream>
 #include <map>
+#include <stdio.h>
```

Comment:
use <cstdio> in c++. All of the c stdlib headers have c++ equivalents that should be used instead.